### YamlMime:PythonClass
uid: azure.servicebus.exceptions.ServiceBusServerBusyError
name: ServiceBusServerBusyError
fullName: azure.servicebus.exceptions.ServiceBusServerBusyError
module: azure.servicebus.exceptions
inheritances:
- azure.servicebus.exceptions.ServiceBusError
summary: 'The Azure Service Bus service reports that it is busy in response to a client
  request to perform an operation.


  Service isn''t able to process the request at this time. Client can wait for a period
  of time,

  then retry the operation.'
constructor:
  syntax: ServiceBusServerBusyError(**kwargs)
methods:
- uid: azure.servicebus.exceptions.ServiceBusServerBusyError.add_note
  name: add_note
  summary: "Exception.add_note(note) \u2013\nadd a note to the exception"
  signature: add_note()
- uid: azure.servicebus.exceptions.ServiceBusServerBusyError.raise_with_traceback
  name: raise_with_traceback
  summary: 'Raise the exception with the existing traceback.


    Deprecated since version 1.22.0: This method is deprecated as we don''t support
    Python 2 anymore. Use raise/from instead.'
  signature: raise_with_traceback()
- uid: azure.servicebus.exceptions.ServiceBusServerBusyError.with_traceback
  name: with_traceback
  summary: "Exception.with_traceback(tb) \u2013\nset self.__traceback__ to tb and\
    \ return self."
  signature: with_traceback()
attributes:
- uid: azure.servicebus.exceptions.ServiceBusServerBusyError.args
  name: args
